src/OrderLinesList/OrderLinesList.js

```diff
@@ -5,7 +5,7 @@ import {
   intlShape,
 } from 'react-intl';
 import PropTypes from 'prop-types';
-import { get } from 'lodash';
+import { get, template } from 'lodash';
```

Comment:
don't forget that it can be broken after changes in CSP

Reply:
and actually you can replace it by regexp

Reply:
this comes from ui-inventory, so I prefer to use similar technique, in case they want to repeat isbn search https://github.com/folio-org/ui-inventory/blob/master/src/Instances.js#L8